crates/freeze/src/datasets/transactions.rs

```diff
@@ -18,20 +24,21 @@ pub struct Transactions {
     gas_limit: Vec<u64>,
     gas_used: Vec<Option<u64>>,
     gas_price: Vec<Option<u64>>,
-    transaction_type: Vec<Option<u32>>,
+    transaction_type: Vec<u32>,
     max_priority_fee_per_gas: Vec<Option<u64>>,
     max_fee_per_gas: Vec<Option<u64>>,
     success: Vec<bool>,
     n_input_bytes: Vec<u32>,
     n_input_zero_bytes: Vec<u32>,
     n_input_nonzero_bytes: Vec<u32>,
-    n_rlp_bytes: Vec<u32>,
+    // not support by alloy
```

Comment:
looks like n_rlp_bytes can be achieved by converting `Transaction` to `Signed` and using `Signed.rlp_encoded_length()`

https://docs.rs/alloy/0.6.4/alloy/consensus/struct.Signed.html#method.rlp_encoded_length